src/cmap/conn/command.rs

```diff
@@ -70,7 +70,6 @@ pub(crate) struct CommandResponse {
 }
 
 impl CommandResponse {
-    #[cfg(test)]
     pub(crate) fn with_document_and_address(source: StreamAddress, doc: Document) -> Self {
```

Comment:
This method is useful in the speculative auth handshake, so I removed the conditional compilation attribute